pygmt/helpers/decorators.py

```diff
@@ -53,6 +53,15 @@
     "W": """\
         pen : str
             Set pen attributes for lines or the outline of symbols.""",
+    "XY": """\
+        xshift : str
+            ``[a|c|f|r][xshift]``.
+            Shift plot origin in x-direction.
+        yshift : str
+            ``[a|c|f|r][yshift]``.
+            Shift plot origin in y-direction. Full documentation is at
+            :gmt-docs:`gmt.html#xy-full`.
+         """,
```

Comment:
> > 1. Do we want a) full documentation from https://docs.generic-mapping-tools.org/6.1/gmt.html#xy-full, or b) just the simplified shortened docsting found at https://docs.generic-mapping-tools.org/6.1/std-opts.html?
> 
> All common aliases (e.g., _region_, _projection_) are using a simplified docstring. We should follow the convention for X and Y, but we may need to find a better way linking to the full documentation.

Ok, sticking with the simplified docstring, and added a link to the full documentation. Below is how it looks like rendered, let me know if we should expand this a little bit more:

![xshift yshift minimal docs with link to gmt manpage](https://user-images.githubusercontent.com/23487320/93729197-93107c80-fc17-11ea-894d-5673305058c3.png)